src/components/documentation/page/page-editormarkdown.html

```diff
@@ -18,7 +18,8 @@
 <div>
     <div class="gv-page-draft-banner" ng-if="!$ctrl.page.published">
         <ng-md-icon icon="warning" class="gv-warning"></ng-md-icon>
-        This page is not published yet and will not be visible to other users
+        {{$ctrl.isMarkdownTemplate() ? 'This page is not available for users yet' :
+        'This page is not published yet and will not be visible to other users' }}
```

Comment:
I wonder if it's not better to move this in the controller to facilitate the merges

Reply:
i agree